iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/Replace3TwoWindingsTransformersByThreeWindingsTransformersTest.java

```diff
@@ -70,6 +71,48 @@ void replaceBasicTest() {
         assertEquals(1, network.getThreeWindingsTransformerCount());
     }
 
+    @Test
+    void replaceSelectedTwoWindingsTransformerOneTest() {
```

Comment:
Could you also add a test with another ID to assert that the replacement is not performed in this case?

Reply:
Done